_quarto.yml

```diff
@@ -20,54 +22,16 @@ book:
     - references.qmd
   appendices:
   - appendix.qmd
+  site-url: https://arrowrbook.com/
+  repo-url: https://github.com/arrowrbook/book/
+  repo-branch: main
+  repo-actions: [source, issue]
```

Comment:
The other option here is `edit` which takes you to the editing view in github. I think we probably don't want that, but having a link to the source and to someone filing an issue seems helpful.

Reply:
nit: how about we move these to the website profile?